### PR TITLE
Feat/get all categories

### LINKED ISSUE #9: feat: impl EventCategory handlers
# Feature
The following EventCategory handlers will be implemented. Use the associated `models` functions-- they have already been implemented.
- [ ] `ListAllCategories` lists all the categories in the database
- [ ] `GetCategory` returns a single category by its id
- [ ] `CreateCategory` creates a new category using the form data
- [ ] `UpdateCategory` updates a category by its id
- [ ] `DeleteCategory` deletes a category by its id

### PR DESCRIPTION
**Pull request by garcialuis**
This PR implements the **GET** `/categories` handler that will allow the client to retrieve the list of event categories. These changes close #9 .
It also includes:
- a migration to see the `user_roles`, `user_status`, `organizations`, and `event_categories` tables with a set of initial values.
- Separates an event date into a `start_date` and `end_date`
- Adds an `is_visible` and `contact_info` fields to the events table.
- Updates error messages so that they are all consistent in using lowercases.

### CODE REVIEW

File: handlers/routes.go [hunk 1]
```diff
@@ -8,10 +8,12 @@ func (s *Server) InitRoutes() {
 	s.Router.HandleFunc("/events/{id}", s.DeleteEvent).Methods("DELETE")
 
 	s.Router.HandleFunc("/categories", s.ListAllCategories).Methods("GET")
-	s.Router.HandleFunc("/categories/{id}", s.GetCategory).Methods("GET")
-	s.Router.HandleFunc("/categories", s.CreateCategory).Methods("POST")
-	s.Router.HandleFunc("/categories/{id}", s.UpdateCategory).Methods("PATCH")
-	s.Router.HandleFunc("/categories/{id}", s.DeleteCategory).Methods("DELETE")
+	/*
```
Comment (villaleo) [inline handlers/routes.go]: Remove commented code

**Review by villaleo — DISMISSED** [review]
Apart from just the commented code, everything looks good! Thanks :)

File: handlers/routes.go (hunk 1 above)
Comment (garcialuis) [reply handlers/routes.go]: Done :)

Comment (garcialuis) [reply handlers/routes.go]: @villaleo I'll need another approval after that last change, please

**Review by villaleo — APPROVED** [review]
Looks good!

**Pull request merged**